node/test_force.js

```diff
@@ -257,8 +257,15 @@ function createCompileApp(tmpDir, appType, os) {
         }
     }
     else {
-        // IOS or Android - Hybrid
-        runProcessCatchError('cordova build', 'COMPILING ' + target, appDir);    
+        if (os == OS.ios) {
+            // IOS - Native
+            runProcessCatchError('cordova build', 'COMPILING ' + target, appDir);    
+        }
+        else {
+            // Android - Native
+            var gradle = isWindows() ? '.\\gradlew.bat' : './gradlew';
```

Comment:
I'm running forcedroid first. This is just to see if the generated app compiles.